wgpu-hal/src/gles/mod.rs

```diff
@@ -366,6 +366,7 @@ impl Texture {
     /// Returns the `target`, whether the image is 3d and whether the image is a cubemap.
     fn get_info_from_desc(desc: &TextureDescriptor) -> u32 {
         match desc.dimension {
+            // TODO: Why `TEXTURE_2D` rather than `TEXTURE_1D`?
```

Comment:
WebGL (1 and 2) as well as some GLES versions do not have 1D textures.
You can replace the TODO comment with the explanation or remove the TODO.